zeppelin-web/webpack.config.js

```diff
@@ -267,9 +267,13 @@ module.exports = function makeWebpackConfig () {
     progress: true,
     contentBase: './src',
     setup: function(app) {
-      app.use(
-        '/bower_components/',
-        require('express').static(path.join(__dirname, './bower_components/')));
+      app.use('**/bower_components/',
+        require('express').static(path.resolve(__dirname, './bower_components/')));
```

Comment:
Thanks @1ambda for review! Yes, it makes sense! Let me updated this part ASAP :)